jgrapht-core/src/main/java/org/jgrapht/graph/AbstractBaseGraph.java

```diff
@@ -39,6 +39,11 @@
  * set ordering (via {@link LinkedHashMap} and {@link LinkedHashSet}). The defaults are reasonable
  * for most use-cases, only change if you know what you are doing.
  *
+ * <p>
+ * For thread-safety information applicable to all subclasses, please see
+ * <a href="https://jgrapht.org/guide/UserOverview#concurrency">the
```

Comment:
Maybe just copy-paste the discussion, to avoid possibly broken links in the future.